src/lib/cherrypickAndCreateTargetPullRequest/getBackportBranchName.ts

```diff
@@ -33,7 +33,8 @@ export function getBackportBranchName({
   const defaultBackportBranchName = 'backport/{{targetBranch}}/{{refValues}}';
 
   const backportBranchName =
-    options.backportBranchName ?? defaultBackportBranchName;
+    (options.backportBranchName && sourcePullRequest?.title) ?? // Empty title resutls in branch name starting with "-" which causes errors
```

Comment:
Maybe instead check the result of template() below and correct if needed while logging a warning.
```ts
const name = template(...)
if (name.startsWith("-")) {
  // ... warn and fix
  return fixedName;
}
return name;
```